src/commcare_cloud/ansible/roles/elasticsearch/tasks/misc_v5.yml

```diff
@@ -0,0 +1,35 @@
+- name: Copy Elasticsearch logging config
+  become: yes
+  copy:
+    src: log4j2.properties
+    dest: "{{ elasticsearch_conf_dir }}/log4j2.properties"
+    owner: elasticsearch
+    group: elasticsearch
+  register: copy_elasticsearch_logging_result
+  tags:
+    - es_conf
+
+- name: Copy Elasticsearch JVM Options
+  become: yes
+  template:
+    src: config/jvm.options.j2
+    dest: "{{ elasticsearch_conf_dir }}/jvm.options"
+    owner: elasticsearch
+    group: elasticsearch
+  register: copy_elasticsearch_logging_result
+  tags:
+    - es_conf
+
+- name: Install elasticsearch python client
+  become: yes
+  pip:
+    name: elasticsearch>=5.0.0,<6.0.0
```

Comment:
Does this use a virtualenv, or does it install globally? What uses this library?

Reply:
I am not sure about it, I kept it because it was there in the previous version and there was a es 7 config that had it too. 
Maybe @dannyroberts or @calellowitz might have some idea where it is used.